perimeterx/README.md

```diff
@@ -0,0 +1,43 @@
+# Agent Check: Perimeterx
+
+## Overview
+
+This check monitors [perimeterx][1].
```

Comment:
```suggestion
This check monitors [PerimeterX][1].
```

Can you provide a brief overview about what PerimeterX is and add a link to the Perimeterx Homepage. It looks like this `[1]` reference is a Datadog docs page.